src/coffea/nanoevents/methods/vector.py

```diff
@@ -669,53 +641,14 @@ def pvec(self):
             behavior=self.behavior,
         )
 
-    @property
-    def rapidity(self):
-        pz = self.z
-        e = self.energy
-        return 0.5 * (numpy.log(e + pz) - numpy.log(e - pz))
-
     @property
     def boostvec(self):
         """The `x`, `y` and `z` components divided by `t` as a `ThreeVector`
 
         This can be used for boosting. For cases where `|t| <= rho`, this
```

Comment:
This is now `r` in the docstring, right?

Reply:
ah yeah.

Reply:
Actually, also need to make sure that the sign in the metric is the same for coffea vs. scikit-hep vector.